sandboxes/js-sandbox/container/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM node:current-alpine
+
```

Кажется неплохо бы обновлять alpine-ские зависимости
```suggestion
RUN apk update && apk upgrade
```